azurelinuxagent/common/protocol/extensions_goal_state.py

```diff
@@ -17,7 +17,7 @@
 # Requires Python 2.6+ and Openssl 1.0+
 import datetime
 
-import azurelinuxagent.common.logger as logger
+from azurelinuxagent.common import logger
```

Comment:
These changes in import statements address new pylint warnings for Python 3.10, there are several of them in the PR